guides/common/modules/proc_using-IdM-credentials-to-log-in-to-the-Project-Hammer-cli.adoc

```diff
@@ -6,10 +6,10 @@ This section describes how to log in to your {Project} Hammer CLI with your {Fre
 .Prerequisites
 * You have enrolled your {ProjectServer} into IdM and configured it to use IdM for authentication.
 For more information, see {InstallingServerDocURL}Using_FreeIPA_{project-context}[Using {FreeIPA}] in _{InstallingServerDocTitle}_.
-* An IdM server is running and reachable.
-* The host is configured to use IdM credentials to log users in to your {Project} Hammer CLI.
+* The host on which you run this procedure is configured to use IdM credentials to log users in to your {Project} Hammer CLI.
 For more information, see {InstallingServerDocURL}Configuring_the_Hammer_CLI_to_Use_FreeIPA_User_Authentication_{project-context}[Configuring the Hammer CLI to Use {FreeIPA} User Authentication] in _{InstallingServerDocTitle}_.
 * The host is an IdM client.
+* An IdM server is running and reachable from the host.
```

Comment:
you say "from the host" here, but "by the host" in two other files. Please consolidate.